### DISCUSSION TITLE
Can't resolve *, but it is there.

### DISCUSSION THREAD

**Discussion by rexkenley** [webpack/webpack#16340] [Q&A] [answered]
I am getting a strange error when generating my code artifacts using webpack. But when I went to look for the "missing" modules, it is there.

Example

ERROR in ../../../node_modules/@babel/runtime-corejs3/regenerator/index.js 3:14-54
Module not found: Error: Can't resolve '../helpers/regeneratorRuntime' in 'C:\Temp10\Ahab\node_modules@babel\runtime-corejs3\regenerator'
@ ./src/lib/dependent-option-sets.ts 2:0-69 105:42-66 108:11-35

![image](https://user-images.githubusercontent.com/10548251/194170591-d30db2ee-c020-4138-8792-987fa0293eab.png)

![image](https://user-images.githubusercontent.com/10548251/194170559-9db002eb-f462-4054-b309-0ba2bc9b8185.png)

**Comment by semiaddict** [comment, marked as answer]
This sometimes happens when two versions of a module are installed, either globally or locally.
Are you sure it's loading the correct module (i.e. from the correct location)?

Why is it loading a module from a Temp folder?

Here are some related issues that might be of some help:

- https://github.com/babel/babel/issues/14576
- https://stackoverflow.com/questions/58919016/babel-and-webpack-are-throwing-cant-resolve-regenerator-runtime-runtime
- https://github.com/bvaughn/react-window/issues/622
- https://bobbyhadz.com/blog/react-module-not-found-cant-resolve-babel-runtime

**Reply by rexkenley** [reply]
THANK YOU @semiaddict 

Adding the extension "js" to the webpack config resolve extensions fixed the issue.
